### DISCUSSION TITLE
#110 Sunrays: funny observation

### DISCUSSION THREAD

**Discussion by gunnarbittersmann** [cssbattle/forum#282] [General]
Chrome (current version 103) on my MacBook renders my solution (100% match) to https://cssbattle.dev/play/110 as:
<img width="400" alt="Bildschirmfoto 2022-07-22 um 08 28 39" src="https://user-images.githubusercontent.com/409844/180377120-4d6a2e82-afef-4522-9c25-19ca471f7f17.png">

Same Chrome version on another MacBook renders all 7 rays.

The difference: first MacBook still runs on macOS Catalina 10.15.7, while the other runs on Monterey 12.4.

Sould the operation system have influence on the rendering in a browser? 🤔

**Comment by SelenIT** [comment]
Haven't you enabled the "Experimental Web Platform features" flag on MacBook?..

**Reply by gunnarbittersmann** [reply]
Stupid me. 🤦‍♂️
Yes, I have Experimental Web Platform features enabled on MacBook 2, but did not have on MacBook 1.
Flag changed – there are ~4 lights~ 7 rays.
I wasn’t aware that we have this particular feature in Chromium on the server, but still behind a flag in Chrome.
